pkg/storage/mvcc_incremental_iterator.go

```diff
@@ -533,6 +533,33 @@ func (i *MVCCIncrementalIterator) NextIgnoringTime() {
 	}
 }
 
+// NextKeyIgnoringTime returns the next distinct key that would be encountered
```

Comment:
should this be in one of the prior commits? I guess waiting to add it here gets it test coverage.

Reply:
I could split it out and add some unit tests. That's probably a good idea anyway, since this could get picked up for production usage somewhere.